### PR TITLE
Update submodules + Regenerate bindings

### PR DESCRIPTION
**Pull request by Beyley**
# Summary of the PR
Updates all our dependencies and regenerates bindings, to prepare for 2.18

### CODE REVIEW

**Comment by Beyley** [comment]
Blocking until upstream merges
https://github.com/gfx-rs/wgpu-native/pull/266
and 
https://github.com/gfx-rs/wgpu-native/pull/272
or similar PRs, these PRs should reduce the amount of wrapper code we'd have to write to get wgpu-native and dawn working cleanly, which isnt really possible right now due to the old header version in use by wgpu-native

This would also allow us to deprecate the Disposal wrapper we have

**Comment by Perksey** [comment]
Holy crap they agreed on refcounting?!?!?!

**Comment by Beyley** [comment]
> Holy crap they agreed on refcounting?!?!?!

ya

**Comment by Beyley** [comment]
Once again blocked until https://github.com/gfx-rs/wgpu-native/pull/277

**Comment by Perksey** [comment]
Can you update Vulkan-Headers/Vulkan-Loader as well please?

**Review by Perksey — COMMENTED** [review]
Approved in principal, just need to make sure there are no updates in VkVideo.

File: src/Microsoft/Silk.NET.Direct2D/Direct2D/Enums/D2D1unnamedEnumAtD2d1h891.gen.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+using System;
+using Silk.NET.Core.Attributes;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.Direct2D
+{
+    [NativeName("Name", "(unnamed enum at d2d1.h:89:1)")]
+    public enum D2D1unnamedEnumAtD2d1h891 : int
```
[the remark this answers is not in the document]
Comment (Perksey) [reply src/Microsoft/Silk.NET.Direct2D/Direct2D/Enums/D2D1unnamedEnumAtD2d1h891.gen.cs]: Old comment, ignore.

File: src/Microsoft/Silk.NET.Direct2D/Direct2D/Structs/D2D1anonymousStructAtCProgramFilesx86WindowsKits10Include100226210umdcommonh2799.gen.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.Direct2D
+{
+    [NativeName("Name", "(anonymous struct at C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\dcommon.h:279:9)")]
+    public unsafe partial struct D2D1anonymousStructAtCProgramFilesx86WindowsKits10Include100226210umdcommonh2799
```
[the remark this answers is not in the document]
Comment (Perksey) [reply src/Microsoft/Silk.NET.Direct2D/Direct2D/Structs/D2D1anonymousStructAtCProgramFilesx86WindowsKits10Include100226210umdcommonh2799.gen.cs]: Old comment, ignore.

**Review by Perksey — APPROVED** [review]

**Comment by Perksey** [comment]
@dotnet-policy-bot rerun

**Pull request merged**